### PR TITLE
frogger-game

### PR DESCRIPTION
**Pull request by Nemooochka**
### Frogger game

[Demo](https://nemooochka.github.io/frontend-nanodegree-arcade-game/index.html) |
[Code base](https://github.com/Nemooochka/frontend-2021-homeworks/blob/frogger-game/submissions/Nemooochka/frogger-game/app.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Nemooochka/frogger-game/app.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+const
+    CELL_WIDTH = 101;
```
Comment (AMashoshyna) [inline submissions/Nemooochka/frogger-game/app.js:2]: Please make formatting consistent.           -__-

File: submissions/Nemooochka/frogger-game/app.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Enemy.prototype.update = function(dt) {
+    this.x += this.speed * dt;
+    if(this.x > FIELD_WIDTH) {
+        this.x = 0;
+        this.speed = this.randomSpeed();
+    }
+    this.checkCollision();
+};
+
+Enemy.prototype.randomSpeed = function() {
```
Comment (AMashoshyna) [inline submissions/Nemooochka/frogger-game/app.js:30]: Usually function/method names contain verbs. This is helpful when you read code and need to understand what happens quickly. How about `getRandomSpeed`?

File: submissions/Nemooochka/frogger-game/app.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+};
+
+Player.prototype.checkBorder = function() {
+    if(this.x < 0) this.x = 0;
+    if(this.x >= FIELD_WIDTH) this.x = FIELD_WIDTH - CELL_WIDTH;
+    if(this.y < 0) this.winGame();
+    if(this.y >= START_POS_Y) this.y = START_POS_Y;
+};
+
+Player.prototype.initialPosition = function() {
```
Comment (AMashoshyna) [inline submissions/Nemooochka/frogger-game/app.js:62]: Same as with `randomSpeed`: would be nice to have a verb in method name. `resetPosition` maybe?

**Review by AMashoshyna — COMMENTED** [review]
Very good! Please fix minor details (see my comments below)

**Comment by Nemooochka** [comment]
Hi @AMashoshyna, thanks for the review!
Made upd, pls re-check :)

**Comment by AMashoshyna** [comment]
Well done!

**Pull request merged**